frontend/src/concepts/topology/PipelineTaskGroupCollapsed.tsx

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
+import ExpandIcon from '@patternfly/react-icons/dist/esm/icons/expand-alt-icon';
+import {
+  RunStatus,
+  LabelPosition,
+  CollapsibleGroupProps,
+  TaskNode,
+  WithSelectionProps,
+  Node,
+  ScaleDetailsLevel,
+  useHover,
+  Layer,
+  DEFAULT_LAYER,
+  TOP_LAYER,
+  NodeModel,
+  observer,
+} from '@patternfly/react-topology';
+import { Icon, Popover } from '@patternfly/react-core';
+import { getNodeStatusIcon } from './utils';
+
+type PipelineTaskGroupCollapsedProps = {
+  children?: React.ReactNode;
+  className?: string;
+  element: Node;
+  hover?: boolean;
+  label?: string; // Defaults to element.getLabel()
+  status?: RunStatus;
+  showStatusState?: boolean;
+  scaleNode?: boolean;
+  hideDetailsAtMedium?: boolean;
+  hiddenDetailsShownStatuses?: RunStatus[];
+  labelPosition?: LabelPosition; // Defaults to bottom
+  badge?: string;
+} & CollapsibleGroupProps &
+  WithSelectionProps;
```

Comment:
Only take the props that are being passed, not need to take those that are never being set.